optuna/visualization/_contour.py

```diff
@@ -66,16 +66,27 @@ def objective(trial):
         params:
             Parameter list to visualize. The default is all parameters.
         target:
-            A function to specify the value to display. If it is :obj:`None`, the objective values
-            are plotted.
+            A function to specify the value to display. If it is :obj:`None` and ``study`` is for
+            single-objective optimization, the objective values are plotted.
+
+            .. note::
+                Specify this argument if ``study`` is for multi-objective optimization.
```

Comment:
A nit but since this seems to be (seems like it will be) reoccurring quite often (and ditto for other occurrences).
```suggestion
                Specify this argument if ``study`` is being used for multi-objective optimization.
```

Reply:
Thanks for the comment. Fixed.